2_cascade/1_prototype-cascade.ipynb

```diff
@@ -0,0 +1,427 @@
+{
+ "cells": [
```

Comment:
I'm not too worried about this notebook yet, but should harmonize what story we're telling with them eventually. (i.e., I'm not reviewing this. you do you)